source/runbooks/disabling-network-connections-if-needed.html.md.erb

```diff
@@ -0,0 +1,106 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disabling network access if required
+last_reviewed_on: 2024-02-14
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+
+There may be occassions where there is a network attack on modernisation platform environments that could require a way of stopping access. This document lists teo possible solutions to the issue.
+
+[NOTE]
+
+> All the steps below assume you can access the system through the AWS console and do not require changes to the terraform code.
+
+## Establish if there is an attack or unwanted access
+
+Establish if there is unwanted access or an attack taking place by checking the details on cloudwatch in both core-network-services and core-logging. It should be obvious from the contents of the logs.
```

Comment:
I'm not sure it is obvious, how would people know what an attack looks like?  Is that something we want to go into here or should this doc be about how to shut down network access rather than how to identify an attack?

Reply:
More about how to stop network access than detailing what constitutes an attack. I couldn't actually find any examples which showed an attack (or unwanted access for that matter) so it's difficult to show examples here.

Reply:
this is still not addressed